src/Shared/Components/ButtonWithSelector/ButtonWithSelector.scss

```
@@ -0,0 +1,5 @@
+.create-button-selector {
+    &:hover, &:active, &:focus {
+        border: 1px solid var(--N200)
```

- Pls check:
```suggestion
        border: 1px solid var(--N200);
```

- Also, the file name should be buttonWithSelector.scss